bark/world/evaluation/rss/safety_polygon.hpp

```diff
@@ -53,6 +53,18 @@ struct SafetyPolygon {
 
 typedef std::shared_ptr<SafetyPolygon> SafetyPolygonPtr;
 
+/**
+ * @brief  Modes for drawing the longitudinal visualization
+ * @note   
+ * @retval None
+ */
+enum LonDirectionMode {
+  AUTO = 0,
```

Comment:
what does auto mean?

Reply:
it either draws the polygon in front or behind based on the other agent. it is stated in the function itself